src/main/webapp/partials/technicalInformationCalculatePointsContribution.jsp

```diff
@@ -0,0 +1,159 @@
+<div id="contentHolder">
+  <br>
+  <ul>
+    <li>
+      <span class="text-bold">General Mechanism</span>
+      <div class="helpSectionContent">
+        Students enter contribution estimates for themselves and their team members using the contribution scale
+        (see <a href="https://github.com/TEAMMATES/teammates/blob/master/docs/glossary.md"> glossary</a>).
+        <br>
+        <br>Based on those values, a student's answer to the following two questions are deduced:
+        <br>(a) In your opinion, what portion of the project did you do?
+        <br>(b) In your opinion, if your teammates are doing the project by themselves without you, how do they compare against each other in terms of contribution?
+        <br>
+        <br>In the calculation, we do not allow (a) to affect (b). We use (b) to calculate the average perceived contribution for each student.
+        <br>
+      </div>
+      <br>
+    </li>
+    <li>
+      <span class="text-bold">Calculation Scheme</span>
+      <div class="helpSectionContent">
+        <ol type="1">
+          <li>
+            <span class="text-bold">Calculate normalizedClaimed values</span>
+            <div class="helpSectionContent">
+              This is required because the total of points entered might not sum up to <code>100 * (team size)</code>.
+              <br>
+              <br>
+              <code>(normalized value) = (original value) * (normalization factor)</code>
+              <br>
+              <br>
+              <div class="bs-example">
+                entered values: 90 [self], 110,130, N/A (total = 330)
+                <br>normalization factor: (100 * 3) / (90 + 110 + 130) = 300 / 330
+                <br>normalized: 82, 100, 118, N/A
+                <br>normalized total = 300 (i.e. 100 * number of inputs)
+              </div>
+              This answers the question (a) above. The student thinks he did 'Equal share - 18%' (as indicated by 82).
+              <br>
+              <br>
+            </div>
+          </li>
+          <li>
+            <span class="text-bold">Calculate peerContributionRatio values by removing self-rating bias</span>
+            <div class="helpSectionContent">
+              Here, we ignore the self rating and normalize remaining values.
+              <br>
+              <br>
+              <div class="bs-example">
+                normalized input (from above): 82,100, 118, N/A
+                <br>Calculating unbiased values:
+                <br>&nbsp;82 → ignored.
+                <br>&nbsp;100 → 100 * 200 / (100 + 118) = 92
+                <br>&nbsp;118 → 118 * 200 / (100 + 118) = 108
+                <br>Unbiased values: [self (ignored)], 92, 108, N/A
+                <br>Unbiased values total = 200 (100 * number of ratings)
+              </div>
+              This answers the question (b) above. In the example above, the student thinks his teammates contribution ratio is 92:108 and is unsure of the third teammate.
```

Comment:
Can also wrap it using example div.

Reply:
Sorry I don't understand what you mean by this part. Right now, this portion looks like this:

![image](https://user-images.githubusercontent.com/24493463/36040217-d31abd44-0dff-11e8-87af-7884b3da8f3b.png)

Reply:
Ohh. Sorry, I have reviewed suing the old commit. :(